src/Groups/GAPGroups.jl

```diff
@@ -296,6 +298,8 @@ function perm(g::PermGroup, L::AbstractVector{<:Base.Integer})
    throw(ArgumentError("the element does not embed in the group"))
 end
 
+perm(g::PermGroup, L::AbstractVector{<:fmpz}) = perm(g, [Int(y) for y in collect(L)])
```

Comment:
This is strange... either I'd drop the `collect`...
```suggestion
perm(g::PermGroup, L::AbstractVector{<:fmpz}) = perm(g, [Int(y) for y in L])
```
or use the two-argument version of `collect`
```suggestion
perm(g::PermGroup, L::AbstractVector{<:fmpz}) = perm(g, collect(Int, L))
```

Reply:
I took the second one.

Reply:
That's not a crash though, right? It just errors out, but Julia keeps running?

This particular error can only be fixed by making `fmpz` a subtype of `Base.Integer` (which currently seems unlikely to happen). 

That said, I am kinda surprised ranges of fmpz's work *at all*, and wonder why and when one would use them? Do we have actual applications, or is this just for the sake of completeness?

Reply:
Another way to fix this particular problem might be to define `axes` for ranges of `fmpz`. 

I've not much personal experience with using `fmpz` ranges, but one use-case would be to call `rand` on them.

Reply:
Sure, but using ranges to define a permutation is fringe, and doing so with fmpz ranges even more so...

Reply:
Yeah, I totally agree that the code here should not be made less efficient/more involved just to accomodate ranges of `fmpz`.

Reply:
It was just for the sake of completeness, but I can just drop the collect.